dotcom-rendering/src/components/CardPicture.tsx

```diff
@@ -31,6 +32,7 @@ type Props = {
  */
 const decideImageWidths = (
 	imageSize: ImageSizeType,
+	aspectRatio: AspectRatio,
```

Comment:
previously we passed aspect ratio a single separate property to fastly. This PR couples the aspect ratio with the breakpoint so that we can vary the aspect ratio between breakpoints. This is currently only required by the immersive feature card